ambient.d.ts

```diff
@@ -499,14 +541,14 @@ declare module '@mojaloop/central-services-stream' {
     }>
   }
 
-  type ConsumeCallback = <T>(error: Error, payload: T) => Promise<void>;
+  type ConsumeCallback<T> = (error: Error | null, payload: T) => Promise<void>;
   type GetMetadataCallback = (err: unknown, result: GetMetadataResult) => void;
 
   namespace Kafka {
     export class Consumer extends EventEmitter {
-      constructor(topics: Array<string>, config: KafkaConsumerConfig)
+      constructor(topics: Array<string>, config: RdKafkaConsumerConfig)
       connect(): Promise<boolean>;
-      consume(consumeCallback: ConsumeCallback): void
+      consume(consumeCallback: ConsumeCallback<any>): void
```

Comment:
let make consume method generic one and correctly propagate the type parameter
```suggestion
      consume<Payload>(consumeCallback: ConsumeCallback<Payload>): void
```